elements/rh-icon/docs/20-guidelines.md

```diff
@@ -0,0 +1,131 @@
+
+<uxdot-example width-adjustment="208px">
+  <img src="../icon-usage-ui-icons.svg" alt="Image of a selection of UI icons">
+</uxdot-example>
+
+<rh-cta href="https://www.redhat.com/en/about/brand/standards/icons/ui-icons">Learn more about UI icons</rh-cta>
+
+### Microns
+
+Microns are UI icons that frequently are used at very small sizes. Using this set, rather than the regular UI icon set, helps to retain the legibility of the larger UI icon.
+
+<uxdot-example width-adjustment="178px">
+  <img src="../icon-usage-microns.svg" alt="Image of a selection of micron icons">
+</uxdot-example>
+
+### Social Icons
+
+These icons are used to represent social media platforms and are often linked.
+
+<uxdot-example width-adjustment="218px">
+  <img src="../icon-usage-social.svg" alt="Image of a selection of social icons">
+</uxdot-example>
+
+## Icons as metaphors
+
+Icons can convey abstract concepts quickly and help users navigate a new interface. For example, a folder icon helps users understand the concept of organizing and storing files in their computer because it provides a connection to file organization in real life. 
+
+The association between an icon and a certain meaning can be strengthened by how frequently it is used in that context. A magnifying glass, for example, is frequently used to denote <strong>Search</strong>, so it is easily recognized by users. This is especially helpful when an icon appears without supporting text nearby.
+
+<uxdot-example color-palette="darkest">
+  <img src="../icon-as-metaphor.png" alt="Image of a search box with an spyglass icon adding additional meaning">
+</uxdot-example>
+
+## Best practices
+
+### Use small sizes
+
+Icons are pictograms and are not meant to be used at large sizes. If you need a visual element that is larger than 100px, use an [illustration](https://www.redhat.com/en/about/brand/standards/illustration) instead.
+
+<div class="grid sm-two-columns">
+  <uxdot-best-practice do>
+    <uxdot-example slot="image" width-adjustment="360px">
+      <img src="../icon-best-practices-size-limit-do.svg" alt="Image of icon at correct size">
+    </uxdot-example>
+    <p>Use icons at 100px or smaller.</p>
+  </uxdot-best-practice>
+
+  <uxdot-best-practice dont>
+    <uxdot-example slot="image" width-adjustment="360px">
+      <img src="../icon-best-practices-size-limit-dont.svg" alt="Image of icon at incorrect size">
+    </uxdot-example>
+    <p>Do not use icons at very large sizes.</p>
+  </uxdot-best-practice>
+</div>
+
+### Ensure consistency
+
+If icons are being used within elements that are grouped together like in a vertical list of links, they should appear to have the same size. The <code>&lt;rh-icon&gt;</code> element maintains a consistent size and ensures everything is aligned when icons are used near other elements.
+
+<div class="grid sm-two-columns">
```

Comment:
Should there be a Do/Dont here about vertical alignment for when an icon has a corresponding text label? eg:

{Icon} Download

Should that alignment be handled by rh-icon (or sibling component)? Or should we place that onus on our users?

Reply:
Icon wouldn't handle alignment here.  The icon doesn't take slotted content. This is more a pattern than something the component should control.

The best practices do/don't would suffice here if there are particular rules around that.   

@marionnegp thoughts?

Reply:
@zeroedin, I'm good with removing "The <code>&lt;rh-icon&gt;</code> element maintains a consistent size and ensures everything is aligned when icons are used near other elements."

Reply:
done